cmd/telemeter-benchmark/main.go

```diff
@@ -62,7 +69,21 @@ func main() {
 	cmd.Flags().StringVar(&opt.Listen, "listen", opt.Listen, "A host:port to listen on for health and metrics.")
 	cmd.Flags().IntVar(&opt.Workers, "workers", opt.Workers, "The number of workers to run in parallel.")
 
+	cmd.Flags().StringVar(&opt.LogLevel, "log-level", opt.LogLevel, "Log filtering level. e.g info, debug, warn, error")
+
+	l := log.NewLogfmtLogger(log.NewSyncWriter(os.Stderr))
+	lvl, err := cmd.Flags().GetString("log-level")
+	if err != nil {
+		level.Error(l).Log("msg", "could not parse log-level.")
+	}
+	l = level.NewFilter(l, logger.LogLevelFromString(lvl))
+	l = log.WithPrefix(l, "ts", log.DefaultTimestampUTC)
+	l = log.WithPrefix(l, "caller", log.Caller(3))
```

Comment:
Why not `log.DefaultCaller`? I just went through the code just to find it's the same value, as I thought this was something special.

Reply:
Yes, it was the default one. I just have forgotten to switch it back after several experiments. I'll fix it right away.